### ISSUE TITLE
sample pipeline jupyter notebook on binder errors-out

### ISSUE THREAD

**Issue by asm582** [project-codeflare/codeflare#12]
**Describe the bug**
sample pipeline jupyter notebook errors out due to undefined variable

**To Reproduce**
1. Go to binder
2. Click on sample pipeline jupyter notebook
3. Run

**Expected behavior**
Jupyter notebook on binder should run without exception

**Additional context**
error while executing cell:
```
pipeline_output = rt.execute_pipeline(pipeline, ExecutionType.FIT, pipeline_input)
node_0_output = pipeline_output.get_xyrefs(node_0)


In [74]:

outputs[0]



---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-74-a45df8d4a457> in <module>
----> 1 outputs[0]

NameError: name 'outputs' is not defined
```

**Comment by chcost** [comment]
@asm582 this issue was fixed with commit db883ecf6a43708dcfec264387d79ea8ef795d87. Can you pull the latest and retry?

**Comment by asm582** [comment]
Interestingly, I thought the binder service always pulls the latest code and runs it. Maybe my service was using stale code from the morning. 

Anyways it works now, closing the issue, thanks.
